app/lib/failure_reasons.rb

```diff
@@ -29,6 +29,8 @@ class FailureReasons
       "teaching_qualifications_from_ineligible_country",
     TEACHING_QUALIFICATIONS_NOT_AT_REQUIRED_LEVEL =
       "teaching_qualifications_not_at_required_level",
+    TEACHING_QUALIFICATIONS_COMPLETED_IN_ELIGIBLE_COUNTRY =
```

Comment:
I'm not sure that we need this line, it'll mean that we end up defining `TEACHING_QUALIFICATIONS_COMPLETED_IN_ELIGIBLE_COUNTRY` as `work_history_duration`.

Reply:
Not sure why i left that in 😆